Jenkinsfile

```diff
@@ -306,17 +306,17 @@ def buildMacOS(workerFunction) {
 
 def buildWindows(nodeVersion, arch) {
   return {
-    myNode('windows && nodejs && cph-windows-01') {
+    myNode('windows && nodejs') {
```

Comment:
I can't remember what the problem was but builds on `cph-windows-01` was failing.